docs/upgrade-to-6.md

```diff
@@ -0,0 +1,649 @@
+# Upgrade to v6
+
+This guide will take you through updating `@gasket/*` packages to `6.x`.
+
+## Update Dependency Versions
+
+Update all `@gasket/` scoped packages to the v6 major version.
+
+This is not an exhaustive list, but rather a sampling of dependencies to
+demonstrate what to look for:
+
+```diff
+"dependencies": {
+-    "@gasket/fetch": "^5.0.2",
++    "@gasket/fetch": "^6.0.0",
+-    "@gasket/intl": "^5.5.0",
++    "@gasket/intl": "^6.0.0",
+-    "@gasket/plugin-workbox": "^5.4.1",
++    "@gasket/plugin-workbox": "^6.0.0",
+}
+```
+
+## Gasket Data
+
+We have decoupled several things from Redux, and instead, have a new construct
+for delivering config data to the client.
+
+If you are using a non-Next.js app, you will need to add the
+[Gasket Data script tag] to be in your document. If you are using a Next.js app,
+we have a [new utility HOC] for which can be used to simplify injecting this
+script tag into the `_document.js` of your app.
+
+### @gasket/data
+
+We have created a new helper package for accessing Gasket Data in the browser.
+This package retrieves the gasketData properties rendered via the aforementioned
+script tag, for use on the client.
+
+_Impacted Plugins/Packages: `@gasket/data`_
+
+### How to Add to Gasket Data
+
+There are two ways to add data to the Gasket Data object:
+
+1. Add to the `res.locals.gasketData` object on the server.
+2. Add to the `public` config property in `app.config.js`
+
+### `public` Config Property
+
+We have created the `public` property in the `app.config.js` file to allow the
+client to access app config properties.
+
+The [@gasket/plugin-config] plugin will return these `public` config properties
+to the browser. The [@gasket/data] package will then access the properties and
+make them available on `.config`.
+
+_Impacted Plugins/Packages: `@gasket/plugin-config`_
+
+### Config Moved to Gasket Data
+
+At this time, the [Intl] and [App config] objects are the only config objects
+that have been moved to the `gasketData` object.
+
+_Impacted Plugins/Packages: `@gasket/plugin-config`, `@gasket/plugin-intl`_
```

Comment:
Let's drop this section. It doesn't actual provide much information and the upgrade details are spelled out elsewhere in the docs